Makefile.am

```diff
@@ -436,6 +436,12 @@ distcheck-hook::
 bots:
 	tools/make-bots
 
+title-case: po/cockpit.pot
+	$(MSGGREP) --no-wrap --msgid -e '\S\+\s' po/cockpit.pot | grep "^msgid " | cut -d " " -f2- | sort --reverse > titles
```

Comment:
This doesn't filter for title cases, just pretty much anything which is not just a single word. A better name would be good to avoid confusion, like "multi-word" or whatever your intention is?

Reply:
oh right, it used to do that but then I just changed it to pick all strings where it might make sense to wonder about "title" vs "sentence" case. (As `VLAN Settings` or `vCPU Settings` or even `$0 Settings` were not picked as title case).

Reply:
changed to `multi-word`